charts/prometheus-pgbouncer-exporter/ci/ci-values.yaml

```diff
@@ -10,7 +10,7 @@ rbac:
   pspEnabled: false
 extraContainers:
   - name: pgbouncer
-    image: bitnami/pgbouncer:latest
+    image: bitnamilegacy/pgbouncer:latest
```

Comment:
bitnamilegacy is gone soon. lets try https://hub.docker.com/r/edoburu/pgbouncer/

Reply:
bitnamilegacy replaces bitnami registry though.

I dont mind either way, whatever you prefer